PHYSICS/AddRope.md

```diff
@@ -15,6 +15,56 @@ Rope does NOT interact with anything you attach it to, in some cases it make int
 Rope will sometimes contract and fall to the ground like you'd expect it to, but since it doesn't interact with the world the effect is just jaring.  
 ```
 
+### Rope Types
+There are 8 different rope types in the base game. Full rope data can be found in `ropedata.xml`.
+- Type: `rage__ropeThin`
+    - Rope Type Value: `0`
+    - Vertices: `1`
+    - Radius: `0.03`
+    - Texture Names: `rope` & `rope_n`
+
+- Type: `rage__ropeWire6`
+    - Rope Type Value: `1`
+    - Vertices: `4`
+    - Radius: `0.015`
+    - Texture Names: `steel_cable` & `steel_cable_n`
+
+- Type: `rage__ropeWire32`
+    - Rope Type Value: `2`
+    - Vertices: `32`
+    - Radius: `0.025`
+    - Texture Names: `steel_cable` & `steel_cable_n`
+
+- Type: `rage__ropeMesh`
+    - Rope Type Value: `3`
+    - Vertices: `6`
+    - Radius: `0.03`
+    - Texture Names: `rope` & `rope_n`
+
+- Type: `rage__ropeThinWire32`
+    - Rope Type Value: `4`
+    - Vertices: `32`
+    - Radius: `0.01`
+    - Texture Names: `rope` & `rope_n`
+
+- Type: `rage__ropeReins`
+    - Rope Type Value: `5`
+    - Vertices: `32`
+    - Radius: `0.005`
+    - Texture Names: `rope` & `rope_n`
+
+- Type: `rage__ropeThin4`
+    - Rope Type Value: `6`
+    - Vertices: `4`
+    - Radius: `0.03`
+    - Texture Names: `rope` & `rope_n`
+
+- Type: `rage__ropeWire64`
+    - Rope Type Value: `7`
+    - Vertices: `64`
+    - Radius: `0.025`
+    - Texture Names: `steel_cable` & `steel_cable_n`
+

```

Comment:
This seems like a lot of unneeded information, this could probably be an enum and still be fine

```c
enum ePhysicsRopeType {
    RopeInvalid = -1,
    RopeThin = 0,
    RopeDefault = 1,
    RopeDefault32 = 2,
    RopeDefaultWire = 3,
    RopeThinWire32 = 4,
    RopeReins = 5,
    RopeThin4 = 6,
}
```

Reply:
Part of the reason that I opted to display the extra info was because unless you go digging in the game files, it's not really found anywhere at all and knowing the amount of verts on a rope and the radius are, in my opinion, some useful info to have here as it will help developers find the rope that suits their needs the best for what they are working on.
With that said, if the standard is to omit this extra detail then I wouldn't be completely against changing to a enum, but I personally think it's important / useful enough info to keep here given the fact that it isn't documented elsewhere.

Reply:
I would generally say that it is unneeded and will clutter the page a lot, you could probably add the amount of verticies the rope uses to the end enum

Reply:
Alright then, thank you for the input and insight into the norms of the natives documentation, I will work on a update later today.